build/openresty/BUILD.openresty.bazel

```diff
@@ -169,7 +169,8 @@ CONFIGURE_OPTIONS = [
     "@kong//:debug_flag": [
         "--with-debug",
         "--with-no-pool-patch",
-        "--with-cc-opt=\"-DNGX_LUA_USE_ASSERT -DNGX_LUA_ABORT_AT_PANIC\"",
+        "--with-cc-opt=\"-g -O0 -DNGX_LUA_USE_ASSERT -DNGX_LUA_ABORT_AT_PANIC\"",
+        "--with-ld-opt=\"-g\"",
```

Comment:
Is it necessary for this PR? should we move these changes to another PR?

Reply:
This is not really needed and can be done in a separate PR if needed.